engines/director/sprite.cpp

```diff
@@ -62,6 +62,8 @@ Sprite::Sprite(Frame *frame) {
 	_moveable = false;
 	_editable = false;
 	_puppet = false;
+	if (g_director->getVersion() >= 600)
```

Comment:
This is incorrect. You already setting this flag in the individual places where we change properties.

With this code, you blindly assign autopuppet to any non-puppet sprite.